### PR TITLE
autotools: remove a few other remaining traces

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

**Comment by liewegas** [comment]
@dachary not sure if the make-debs.sh script changes are correct.  what actually uses that script though?

**Comment by dachary** [comment]
@liewegas a few people use it apparently but nothing in the teuthology machinery AFAIK

File: make-debs.sh [hunk 1]
```diff
@@ -34,19 +34,7 @@ git clean -dxf
...
-./autogen.sh
-./configure --with-rocksdb --with-ocf \
-    --with-nss --with-debug --enable-cephfs-java \
-    --with-lttng --with-babeltrace
-#
-# use distdir= to set the name of the top level directory of the
-# tarbal to match the desired version
-#
-make distdir=ceph-$vers dist
+./make-dist $vers
```
Comment (tchaikov) [inline make-debs.sh]: the output file generated by `make-dist` is named `ceph-$ver.tar.bz2`. we might need to update following command lines accordingly:
```sh
mv ceph-$vers.tar.bz2 $releasedir/ceph_$vers.orig.tar.bz2
tar -C $releasedir -jxf $releasedir/ceph_$vers.orig.tar.bz2
```

File: run-make-check.sh [hunk 2]
```diff
@@ -65,19 +65,10 @@ function run() {
...
-        $DRY_RUN ctest $CHECK_MAKEOPTS --output-on-failure || return 1
-    else
-        $DRY_RUN ./autogen.sh || return 1
-        $DRY_RUN ./configure "$@"  --with-librocksdb-static --disable-static --with-radosgw --with-debug --without-lttng \
-            CC="ccache gcc" CXX="ccache g++" CFLAGS="-Wall -g" CXXFLAGS="-Wall -g" || return 1
-        $DRY_RUN make $BUILD_MAKEOPTS || return 1
-        $DRY_RUN make $CHECK_MAKEOPTS check || return 1
-        $DRY_RUN make dist || return 1
-    fi
+    $DRY_RUN ./do_cmake.sh $@ || return 1
```
Comment (tchaikov) [inline run-make-check.sh]: or we can just remove this script, and rename `run-cmake-check.sh` to `run-make-check.sh`?

**Comment by liewegas** [comment]
@tchaikov ok to merge?

File: make-debs.sh (hunk 1 above)
Comment (tchaikov) [inline make-debs.sh]: @liewegas this comment is not addressed.

**Comment by liewegas** [comment]
oops, fixed

**Pull request merged**